siriuspy/siriuspy/ramp/conn.py

```diff
@@ -1,85 +1,337 @@
+        c = ConnTiming.Const
+        readbacks = dict()
+        readbacks[c.EVG_Evt01Mode] = c.MODE_EXTERNAL
+        readbacks[c.EVR1_OTP08Pulses] = 1
+        return self._check(readbacks)
+
+    # --- private methods ---
+
+    def _define_properties(self, prefix, connection_callback, callback):
+        c = ConnTiming.Const
+        p = prefix
+        properties = (
+            _EpicsProperty(c.EVG_ContinuousEvt, '-Sel', '-Sts', p,
+                           c.STATE_DISBL,
+                           connection_callback=connection_callback,
+                           callback=callback),
+            _EpicsProperty(c.EVG_DevEnbl, '-Sel', '-Sts', p, c.STATE_ENBL,
+                           connection_callback=connection_callback,
+                           callback=callback),
+            _EpicsProperty(c.EVG_ACEnbl, '-Sel', '-Sts', p, c.STATE_ENBL,
+                           connection_callback=connection_callback,
+                           callback=callback),
+            _EpicsProperty(c.EVG_Evt01Mode, '-Sel', '-Sts', p,
+                           c.MODE_EXTERNAL,
+                           connection_callback=connection_callback,
+                           callback=callback),
+            _EpicsProperty(c.EVG_Evt01ExtTrig, '-Cmd', '-Cmd', p, None,
+                           connection_callback=connection_callback,
+                           callback=callback),
+            _EpicsProperty(c.EVR1_DevEnbl, '-Sel', '-Sts', p, c.STATE_ENBL,
+                           connection_callback=connection_callback,
+                           callback=callback),
+            _EpicsProperty(c.EVR1_OTP08State, '-Sel', '-Sts', p, c.STATE_ENBL,
+                           connection_callback=connection_callback,
+                           callback=callback),
+            _EpicsProperty(c.EVR1_OTP08Polarity, '-Sel', '-Sts',
+                           p, c.STATE_NORMAL,
+                           connection_callback=connection_callback,
+                           callback=callback),
+            _EpicsProperty(c.EVG_ACDiv, '-SP', '-RB', p, 30,
+                           connection_callback=connection_callback,
+                           callback=callback),
+            _EpicsProperty(c.EVG_RFDiv, '-SP', '-RB', p, 4,
+                           connection_callback=connection_callback,
+                           callback=callback),
+            _EpicsProperty(c.EVR1_OTP08Width, '-SP', '-RB', p, 7000,
+                           connection_callback=connection_callback,
+                           callback=callback),
+            _EpicsProperty(c.EVR1_OTP08Evt, '-SP', '-RB', p, 1,
+                           connection_callback=connection_callback,
+                           callback=callback),
+            _EpicsProperty(c.EVR1_OTP08Pulses, '-SP', '-RB', p, _MAX_WFMSIZE,
+                           connection_callback=connection_callback,
+                           callback=callback),)
+        super().__init__(properties)
```

Comment:
Wouldn't it be better to return the variable properties and call the `super().__init__` method from inside `self.__init__`